packages/astro/src/cli/index.ts

```diff
@@ -49,7 +49,6 @@ function printAstroHelp() {
 			['--config <path>', 'Specify the path to the Astro config file.'],
 			['--root <path>', 'Specify the path to the project root folder.'],
 			['--legacy-build', 'Use the build strategy prior to 0.24.0'],
-			['--experimental-ssr', 'Enable SSR compilation fot 3rd-party adapters.'],
```

Comment:
Oops, it seems that some unwanted changes got leaked

Reply:
Oh, sorry, I didn't know about it

Reply:
No problem! I should have added a comment here for reviewers